apps.toml

```
@@ -2540,7 +2541,7 @@ url = "https://github.com/YunoHost-Apps/photonix_ynh"
 
 [photoprism]
 category = "multimedia"
-level = 7
+level = 0
```

```suggestion
level = 7
```

(apparently photoprism is back to level 7 in latest job :tada: )